pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -293,12 +293,12 @@ func TestPriorityQueue_AddUnschedulableIfNotPresent(t *testing.T) {
 	}
 }
 
-// TestPriorityQueue_AddUnschedulableIfNotPresent_Backoff tests scenario when
-// AddUnschedulableIfNotPresent is called asynchronously pods in and before
-// current scheduling cycle will be put back to activeQueue if we were trying
-// to schedule them when we received move request.
+// TestPriorityQueue_AddUnschedulableIfNotPresent_Backoff tests the scenarios when
+// AddUnschedulableIfNotPresent is called asynchronously.
+// Pods in and before current scheduling cycle will be put back to activeQueue
```

Comment:
we seem to be testing that they go back to the backoff queue. Maybe we should also "forward" the time and make sure they make it back to the activeQueue?

Reply:
We can. But in this PR I just want to focus on resolving the racing issue :)

Reply:
sg